org-node/src/client.rs

```diff
@@ -235,11 +285,200 @@ impl Client {
         }
     }
 
+    /// Track peers that are found by the node that are interested in shared projects;
+    /// This process is spawned within `Client::run` and is run indefinitely.
+    async fn track_peers(mut peer_rx: mpsc::Receiver<ProjectPeerTransmitter>) {
+        tracing::info!(target: "org-node", "Spawning Track Peers Listener");
+        while let Some(rx) = peer_rx.recv().await {
+            let api = rx.0.api;
+            let urn = rx.0.urn;
+            let peer_info = rx.0.peer_info;
+            let paths = rx.0.paths;
+
+            let ack = rx.1;
+
+            match Client::track_project(&api, &urn, &peer_info).await {
+                Err(err) => {
+                    tracing::error!(target: "org-node", "Error tracking {}: {}", urn, err);
```

Comment:
We don't return any error back, so how does the sender know to retry?

Reply:
Thanks, nice catch! 5dfbfd6 adds.